bin/parse-server

```diff
@@ -10,26 +10,28 @@ if (process.env.PARSE_SERVER_OPTIONS) {
 	options = JSON.parse(process.env.PARSE_SERVER_OPTIONS);
 
 } else {
-
-	options.databaseURI = process.env.PARSE_SERVER_DATABASE_URI;
-	options.cloud = process.env.PARSE_SERVER_CLOUD_CODE_MAIN;
-	options.collectionPrefix = process.env.PARSE_SERVER_COLLECTION_PREFIX;
+  options.app                        = {};
```

Comment:
tbh, singletons have given me a real pain to make the multiple apps work correctly, the first in line being Parse, because everything is asynchronous, you can't guarantee that your singleton didn't change it's state between 2 callbacks :)